test/units/module_utils/facts/tests_date_time.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+import datetime
+
+from ansible.module_utils.facts.system.date_time import DateTimeFactCollector
+
+TIMESTAMP = datetime.datetime(2020, 7, 11, 12, 34, 56, 124356)
+
+
+@pytest.fixture
+def fake_now(monkeypatch):
+
+    class FakeNow:
+        @classmethod
+        def now(cls):
+            return TIMESTAMP
+
+    monkeypatch.setattr(datetime, 'datetime', FakeNow)
+
+
+@pytest.fixture
+def date_collector(monkeypatch, fake_now):
```

Comment:
When you don't refer to the fixture return value, it's better to inject it via `usefixtures` so that there's no unused argument in the function:
```suggestion
@pytest.mark.usefixtures('fake_now')
def date_collector(monkeypatch):
```

Reply:
Thank you. Changed a name of the function. But `@pytest.mark.usefixtures('fake_now')` didn't mock the `datetime.datetime.now()` for the function.

Reply:
Ah, my bad. This doesn't work on fixtures.